
  1) test get_payments_with_last_done_state/1 returns payments done occured during passed duration (Omc.PaymentsTest)
     test/omc/payments_test.exs:240
     ** (ArgumentError) cannot allow #PID<0.556.0> to use Omc.PaymentProviderOxapayMock from #PID<0.2025.0> because it is already 
allowed by #PID<0.2021.0>.
     
     If you are seeing this error message, it is because you are either setting up allowances from different processes or your tes
ts have async: true and you found a race condition where two different tests are allowing the same process
     
     code: payment_state_by_callback_fixture(payment_request, :done)
     stacktrace:
       (mox 1.0.2) lib/mox.ex:683: Mox.allow/3
       (omc 0.1.0) test/support/fixtures/payment_fixtures.ex:44: Omc.PaymentFixtures.payment_state_by_callback_fixture/2
       test/omc/payments_test.exs:243: (test)



  2) test get_payments_with_last_done_state/1 payments which their state is update befor duration should not listed (Omc.PaymentsT
est)
     test/omc/payments_test.exs:247
     ** (ArgumentError) cannot allow #PID<0.556.0> to use Omc.PaymentProviderOxapayMock from #PID<0.2029.0> because it is already 
allowed by #PID<0.2021.0>.
     
     If you are seeing this error message, it is because you are either setting up allowances from different processes or your tes
ts have async: true and you found a race condition where two different tests are allowing the same process
     
     code: payment_state_by_callback_fixture(payment_request, :done)
     stacktrace:
       (mox 1.0.2) lib/mox.ex:683: Mox.allow/3
       (omc 0.1.0) test/support/fixtures/payment_fixtures.ex:44: Omc.PaymentFixtures.payment_state_by_callback_fixture/2
       test/omc/payments_test.exs:250: (test)

..........................
Finished in 7.9 seconds (7.2s async, 0.6s sync)
263 tests, 2 failures

Randomized with seed 20519
